R/doc-data.R

```diff
@@ -1,4 +1,12 @@
-utils::globalVariables(c("world_geocoder", "esri_wkids"))
+# Make world_geocoder and esri_wkids available on load
+.onLoad <- function(lib, pkg) {
+  utils::data(
+    list = c("world_geocoder", "esri_wkids"),
+    package = pkg,
+    envir = parent.env(environment())
+  )
+}
+
```

Comment:
interesting approach! Can we move this to `zzz.R` to be consistent with other uses of `.onLoad()`?

Reply:
Huh, so I learned this approach a couple years ago but, while looking for a reference so I could confirm this is the "standard" approach, I found out that it actually isn't the best practice https://r-pkgs.org/data.html#sec-data-data

I'll swap this for a `arcgisgeocode::` prefix for the datasets which is the recommended approach in the R Packages book.